server/src/types/grades.ts

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+export interface Grade {
```

Comment:
Could have a comment for why this is separate from `AttainmentGrade`, and maybe a more descriptive name of the purpose of this interface.

Reply:
Actually after thinking about this, we don't need the `Grade` interface. Refactored so that only `AttainmentGrade` used.